discordrp_mpris/__main__.py

```diff
@@ -218,8 +223,17 @@ def build_replacements(self, player: Player, metadata) -> Dict[str, Optional[str
                 replacements[key] = source
             else:
                 replacements[key] = " & ".join(source)
+            replacements[key] = shorten(
+                replacements[key],
+                width=self.config.player_get(player, 'max_artist_len', 32),
+                placeholder="..."
```

Comment:
If we use `…` here (and in the other places), we can save some characters. Discord should be able to handle it.

Reply:
Done.